insights_content_template_renderer/utils.py

```diff
@@ -87,7 +87,13 @@ def get_template_function(template_name, template_text, report: Report):
     log.debug(template_text)
 
     template = renderer.template(escape_raw_text_for_js(template_text), DoT_settings)
-    return js2py.eval_js(template)
+    try:
+        return js2py.eval_js(template)
+    except Exception as ex:
```

Comment:
couldn't we just check for the `js2py.internals.simplex.JsException` we're expecting?

Reply:
done :)